tests/views_integration/Tribe/Events/Views/V2/Template/__snapshots__/TitleTest__test_day_view__1.php

```diff
@@ -1 +1 @@
-<?php return 'Upcoming Events';
+<?php return 'Events for August 12, 2020 - August 12, 2020';
```

Comment:
This is the date of the snapshot generation, this means some date-related value is not mocked correctly.
This must be fixed.

Reply:
This text looks wrong as well?